src/domainModules/collectionMammals/components/MammalForm/transformations/output.js

```diff
@@ -1,5 +1,27 @@
 import immutable from 'object-path-immutable'
 
+export const getFeatureObservationTypeId = ({ featureObservationTypeName }) => {
+  switch (featureObservationTypeName) {
+    case 'age': {
+      return 1
+    }
+    case 'ageStage': {
+      return 2
+    }
+    case 'conditionAtCollecting': {
+      return 3
+    }
+    case 'sex': {
+      return 4
+    }
```

Comment:
these IDs don't match the DB. the DB seems to have more types than the frontend and different IDs. check with Ida
<img width="536" alt="screenshot 2018-01-21 18 35 42" src="https://user-images.githubusercontent.com/12738484/35196996-f996f4d6-fed9-11e7-9af6-aee9fdd3e02f.png">

Reply:
I have sent the question to Ida